src/test/ui/issues/issue-34721.stderr

```diff
@@ -5,14 +5,24 @@ LL |     pub fn baz<T: Foo>(x: T) -> T {
    |                        - move occurs because `x` has type `T`, which does not implement the `Copy` trait
 LL |         if 0 == 1 {
 LL |             bar::bar(x.zero())
-   |                      - value moved here
+   |                      -------- value moved due to this method call
 LL |         } else {
 LL |             x.zero()
-   |             - value moved here
+   |             -------- value moved due to this method call
 LL |         };
 LL |         x.zero()
    |         ^ value used here after move
    |
+note: this function takes `self`, which moves the receiver
+  --> $DIR/issue-34721.rs:4:5
+   |
+LL |     fn zero(self) -> Self;
+   |     ^^^^^^^^^^^^^^^^^^^^^^
+note: this function takes `self`, which moves the receiver
```

Comment:
hmm, I wonder if this could get verbose. Can we check for cases where we've already printed the method once and suppress it? Not sure how hard that would be.